terraform/projects/infra-security-groups/ci-master.tf

```
@@ -92,7 +92,7 @@ resource "aws_security_group_rule" "ci-master-elb_ingress_github_https" {
   protocol  = "tcp"
 
   security_group_id = "${aws_security_group.ci-master_elb.id}"
-  cidr_blocks       = ["${data.github_ip_ranges.github.hooks}"]
+  cidr_blocks       = ["${data.github_ip_ranges.github.hooks_ipv4}"]
```

```suggestion
  cidr_blocks      = ["${data.github_ip_ranges.github.hooks_ipv4}"]
  ipv6_cidr_blocks = ["${data.github_ip_ranges.github.hooks_ipv6}"]
```

From [this doc](https://github.com/integrations/terraform-provider-github/blob/8a197474b6f5ec265b6ad426c0ed11cc25ffa8d8/website/docs/d/ip_ranges.html.markdown#attributes-reference) the `hooks_ipv4` returns a subset, but I suspect we want _all_ IPs in case some of the servers use an IPv6 address.

Reference: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group_rule#example-usage

---

I think this is fine, IPv6 is not enable on gov.uk as far as I know.